testsuite/integration/elytron-oidc-client/src/test/java/org/wildfly/test/integration/elytron/oidc/client/deployment/OidcWithDeploymentConfigTest.java

```diff
@@ -567,6 +618,17 @@ public void testCorsRequestWithEnableCorsWithInvalidOrigin() throws Exception {
         }
     }
 
+    @Test
+    @InSequence(27)
+    public void testFormWithOidc() throws Exception {
+        try {
+            deployer.deploy(FORM_WITH_OIDC_EAR_APP);
+            super.testFormWithOidc();
+        } finally {
+            deployer.undeploy(FORM_WITH_OIDC_EAR_APP);
+        }
+    }
+
```

Comment:
Maybe it would be good to also add some checks where the authentication fails? (incorrect username/passwords/roles, etc)

Reply:
for OIDC login I am using the common method loginToApp which is
      doing all the checking and reporting for the other tests.
    for testFormCredentials() that type of detail is not return in the
      response. Only the HTTP failure number is returned.

Reply:
I see there's an error.jsp page configured for the form app. Would it be possible to check that we get to the error page when an invalid username/password is specified (similar to the test here: https://github.com/wildfly/wildfly/blob/main/testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/http/FormMechTestBase.java#L142-L151)?

Same for OIDC, would it be possible to check that the wrong password is handled correctly as shown here: https://github.com/wildfly/wildfly/blob/main/testsuite/integration/elytron-oidc-client/src/test/java/org/wildfly/test/integration/elytron/oidc/client/OidcBaseTest.java#L157-L159

Reply:
done